Signal/src/AppDelegate.m

```diff
@@ -1180,6 +1180,8 @@ - (void)checkIfAppIsReady
     [self ensureRootViewController];
 
     [OWSBackup.sharedManager setup];
+
+    [OWSBackupLazyRestoreJob run];
```

Comment:
On app launch we try to resume lazy restore.

Reply:
runAsync?

Reply:
Done.